editor/components/block-inspector/index.js

```diff
@@ -2,22 +2,59 @@
  * External dependencies
  */
 import { connect } from 'react-redux';
+import { uniq, keys, flatten } from 'lodash';
 
 /**
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
 import { Slot } from '@wordpress/components';
+import { getBlockType } from '@wordpress/blocks';
 
 /**
  * Internal Dependencies
  */
 import './style.scss';
-import { getSelectedBlock, getSelectedBlockCount } from '../../selectors';
+import { getSelectedBlock, getSelectedBlockCount, getMultiSelectedBlocks } from '../../selectors';
 
-const BlockInspector = ( { selectedBlock, count } ) => {
+const BlockInspector = ( { selectedBlock, count, multiSelectedBlocks, onChange } ) => {
 	if ( count > 1 ) {
-		return <span className="editor-block-inspector__multi-blocks">{ __( 'Coming Soon' ) }</span>;
+		const names = uniq( multiSelectedBlocks.map( ( { name } ) => name ) );
+
+		if ( names.length === 1 ) {
+			const Inspector = getBlockType( names[ 0 ] ).inspector;
+
+			if ( ! Inspector ) {
+				return null;
+			}
+
+			const attributeArray = multiSelectedBlocks.map( ( block ) => block.attributes );
+			const attributeKeys = uniq( flatten( attributeArray.map( keys ) ) );
+			const attributes = attributeKeys.reduce( ( acc, key ) => {
```

Comment:
Attempted to use lodash's `intersection` or similar, but not quite sure how to get it to work.